themes/default/html/projections.html

```diff
@@ -28,9 +28,11 @@ <h4><small>Revised Target<br>{{ revised.percent_str }}%</small></h4>
         </div>
     </div>
 
-<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.3/Chart.min.js"></script>
+<!-- Graphs -->
+<!-- Cant move this yet - it needs to be loaded before our chart script
```

Comment:
Out of date now?

Reply:
No, this is (I believe) referring to moving ths <script></script> blocks to their own files and loading them from the respective theme files